src/locales/en.json

```diff
@@ -623,6 +623,11 @@
   "mno_enterprise.templates.dashboard.organization.credit_card.billing_form.postcode": "Postcode",
   "mno_enterprise.templates.dashboard.organization.credit_card.billing_form.country": "Country",
   "mno_enterprise.templates.dashboard.organization.credit_card.billing_form.country_select": "-- SELECT --",
+  "mno_enterprise.templates.dashboard.organization.credit_card.confirm_modal.cancel": "Cancel",
+  "mno_enterprise.templates.dashboard.organization.credit_card.confirm_modal.confirm": "Ok",
+  "mno_enterprise.templates.dashboard.organization.credit_card.confirm_modal.header": "<b>Payment terms</b>",
+  "mno_enterprise.templates.dashboard.organization.credit_card.confirm_modal.disclaimer": "<span class='text-danger'>Please note: This credit card will be used to process all outstanding invoices automatically on the 2nd of each month. Click ok to continue.</span>",
```

Comment:
The span tag should be in the template not in the locale

Reply:
Updated the component to allow dynamic classes and removed span tags from the locale. 👍